src/lib/add-on/clients/add-on_auto.rb

```diff
@@ -19,6 +19,8 @@
 
 require "yast"
 require "installation/auto_client"
+require "uri"
+require "yast2/rel_url"
```

Comment:
It uses `AddOnProduct.GetAbsoluteURL` so this `require` should not be needed...

Reply:
That's right.